src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -57,6 +58,9 @@ MapDocumentActionHandler::MapDocumentActionHandler(QObject *parent)
     Q_ASSERT(!mInstance);
     mInstance = this;
 
+    mActionGotoTile = new QAction(this);
+    mActionGotoTile->setShortcut(tr("Ctrl+G"));
```

Comment:
The Ctrl+G shortcut is taken for toggling the grid. However, I think Ctrl+G is a better suit for "go to" and instead the grid toggle could be changed. Maybe `#` works for that, which is what Inkscape uses.